pandas/tests/indexes/interval/test_interval.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
 from pandas import (
     Interval, IntervalIndex, Index, isna, notna, interval_range, Timestamp,
```

Comment:
I believe your test for ``.length`` should change? IIRC you were catching he exception

Reply:
`.length` had two tests: `test_length` and `test_length_errors`.  I've removed `test_length_errors` in full, since we should be prohibiting dtypes that would cause exceptions. I've left `test_length` unchanged since it was only testing valid dtypes.